# Warp
# Origin date: Tue, 12 Nov 2019 15:31:14 -0800
# Local date: Tue, 12 Nov 2019 15:31:14 -0800
# Commit hash: 06ab50e
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warpCparallel.cpython-37m-x86_64-linux-gnu.so
# Thu Jan  2 15:16:50 2020
# 24 processors
# import warp time 2.151589870452881 seconds
# For more help, type warphelp()
--------------------------------------------------------------------------
An MPI process has executed an operation involving a call to the
"fork()" system call to create a child process.  Open MPI is currently
operating in a condition that could result in memory corruption or
other system errors; your MPI job may hang, crash, or produce silent
data corruption.  The use of fork() (or system() or other calls that
create child processes) is strongly discouraged.  

The process that invoked fork was:

  Local host:          eu-a6-012-07 (PID 12936)
  MPI_COMM_WORLD rank: 22

If you are *absolutely sure* that your application will successfully
and correctly survive a call to fork(), you may disable this warning
by setting the mpi_warn_on_fork MCA parameter to 0.
--------------------------------------------------------------------------
('l_fftw_fort', False)
WARNING: module txphysics is not accessible.
WARNING: module desorb is not accessible.
0.0007224415584921976
Simulating ECLOUD in a crab cavity
[eu-a6-012-07:12910] 23 more processes have sent help message help-mpi-runtime.txt / mpi_init:warn-fork
[eu-a6-012-07:12910] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
Using cosine_3D emission angle distribution.
Secondary emission model: ECLOUD E0=150.0000 s=1.3500
#############################################################
Temp distribution found. Reloading it as initial distribution
#############################################################
Plot file name complete_sim.005.cgm
 ***  particle simulation package W3D generating
 ---  Resetting lattice array sizes
 ---  Allocating space for particles
 ---  Loading particles
 ---  Setting charge density
 ---  done
 ---  Allocating Win_Moments
 ---  Allocating Z_Moments
 ---  Allocating Lab_Moments
 Atomic number of ion =  1.0073E+00
 Charge state of ion  =  1.0000E+00
 Initial X, Y emittances =  0.0000E+00,   0.0000E+00 m-rad
 Initial X,Y envelope radii  =  0.0000E+00,   0.0000E+00 m
 Initial X,Y envelope angles =  0.0000E+00,   0.0000E+00 rad
 Input beam current =  0.0000E+00 amps
 Current density =  0.0000E+00 amps/m**2
 Charge density =  0.0000E+00 Coul/m**3
 Number density =  0.0000E+00
 Plasma frequency     =  0.0000E+00 1/s
    times dt          =  0.0000E+00
    times quad period =  0.0000E+00
 Plasma period        =  6.2832E+36 s
 X-, Y-Thermal Velocities     =  0.0000E+00,   0.0000E+00 m/s
    times dt                  =  0.0000E+00,   0.0000E+00 m
    times dt/dx, dt/dy (X, Y) =  0.0000E+00,   0.0000E+00
 X-, Y-Debye Wavelengths  =  0.0000E+00,   0.0000E+00 m
    over dx, dy (X and Y) =  0.0000E+00,   0.0000E+00
 Longitudinal thermal velocity (rms) =  0.0000E+00 m/s
    times dt                   =  0.0000E+00 m
    times dt/dz                =  0.0000E+00
 Longitudinal Debye wavelength =  0.0000E+00 m
    over dz                    =  0.0000E+00
 Beam velocity =  0.0000E+00 m/s
    over c     =  0.0000E+00
 Kinetic energy =  0.0000E+00 eV
 Weight of simulation particles =  1.0000E+00
 Number of simulation particles =            0
 Number of real particles =  0.0000E+00
 Total mass =  0.0000E+00 kg
 Total charge =  0.0000E+00 Coul
 Generalized perveance =  0.0000E+00
 Characteristic current =  3.1297E+07 amps
 Budker parameter =  0.0000E+00
 Timestep size dt =  2.5000E-11 s
 Tune length =  0.0000E+00
 Undep. X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Undep. X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 Undep. X-, Y-Betatron wavelengths  =  0.0000E+00,   0.0000E+00 m
 Dep.   X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Dep.   X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 X-, Y-Tune Depressions (sigma/sigma0) =  0.0000E+00,   0.0000E+00
 Space charge wave velocity =  0.0000E+00 m/s
 Effective wall radius =  2.1355E-01 m
 Geometric factor =  7.9805E+01
 X-, Y-Emittance over Space charge forces =  0.0000E+00,   0.0000E+00
 Number of grid points in x =      200
 Number of grid points in y =      200
 Number of grid points in z =      100
 Grid spacing in x =  1.5100E-03 m
 Grid spacing in y =  1.2200E-03 m
 Grid spacing in z =  6.0200E-03 m
 Bend radius =  0.0000E+00 m
 Bending field =  0.0000E+00 T
 Bend length =  0.0000E+00 m
 Straight section length =  0.0000E+00 m
 Z at start of first bend =  0.0000E+00 m
it =      0 time = 0.0000E+00 pz = 0.0000E+00 ese = 0.0000E+00 ek = 0.0000E+00 et = 0.0000E+00
 ***  particle simulation package W3D running
Multigrid: Error converged to   2.796E-07 in     8 v-cycles             
it =      1 time = 2.5000E-11 pz = 0.0000E+00 ese = 7.1219E-12 ek = 0.0000E+00 et = 7.1219E-12
Multigrid: Error converged to   5.120E-07 in    13 v-cycles             
it =      2 time = 5.0000E-11 pz = 0.0000E+00 ese = 3.0800E-11 ek = 0.0000E+00 et = 3.0800E-11
Saving a checkpoint!
===========================
Bunch passage: 1
Number of electrons in the dipole: 20000895
Number of macroparticles: 4200188
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 2
Number of electrons in the dipole: 18708625
Number of macroparticles: 4028862
Previous passage took 1677s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 3
Number of electrons in the dipole: 20138465
Number of macroparticles: 4277405
Previous passage took 1759s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 4
Number of electrons in the dipole: 22209366
Number of macroparticles: 4769415
Previous passage took 1870s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 5
Number of electrons in the dipole: 25180667
Number of macroparticles: 5434702
Previous passage took 1994s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 6
Number of electrons in the dipole: 29615949
Number of macroparticles: 6360144
Previous passage took 2156s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 7
Number of electrons in the dipole: 36050679
Number of macroparticles: 7820969
Previous passage took 2377s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 7846514
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py:62: UserWarning: there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics
  warnings.warn("there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics")
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 251, in step
    self.secelec.wspecies, sec)
NameError: name 'sec' is not defined
--------------------------------------------------------------------------
mpirun noticed that the job aborted, but has no info as to the process
that caused that situation.
--------------------------------------------------------------------------
# Warp
# Origin date: Tue, 12 Nov 2019 15:31:14 -0800
# Local date: Tue, 12 Nov 2019 15:31:14 -0800
# Commit hash: 06ab50e
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warpCparallel.cpython-37m-x86_64-linux-gnu.so
# Thu Jan  2 19:46:22 2020
# 24 processors
# import warp time 2.293361186981201 seconds
# For more help, type warphelp()
--------------------------------------------------------------------------
An MPI process has executed an operation involving a call to the
"fork()" system call to create a child process.  Open MPI is currently
operating in a condition that could result in memory corruption or
other system errors; your MPI job may hang, crash, or produce silent
data corruption.  The use of fork() (or system() or other calls that
create child processes) is strongly discouraged.  

The process that invoked fork was:

  Local host:          eu-a6-001-07 (PID 67087)
  MPI_COMM_WORLD rank: 2

If you are *absolutely sure* that your application will successfully
and correctly survive a call to fork(), you may disable this warning
by setting the mpi_warn_on_fork MCA parameter to 0.
--------------------------------------------------------------------------
('l_fftw_fort', False)
WARNING: module txphysics is not accessible.
WARNING: module desorb is not accessible.
0.0007224415584921976
Simulating ECLOUD in a crab cavity
[eu-a6-001-07:67080] 23 more processes have sent help message help-mpi-runtime.txt / mpi_init:warn-fork
[eu-a6-001-07:67080] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
Using cosine_3D emission angle distribution.
Secondary emission model: ECLOUD E0=150.0000 s=1.3500
#############################################################
Temp distribution found. Reloading it as initial distribution
#############################################################
Plot file name complete_sim.007.cgm
 ***  particle simulation package W3D generating
 ---  Resetting lattice array sizes
 ---  Allocating space for particles
 ---  Loading particles
 ---  Setting charge density
 ---  done
 ---  Allocating Win_Moments
 ---  Allocating Z_Moments
 ---  Allocating Lab_Moments
 Atomic number of ion =  1.0073E+00
 Charge state of ion  =  1.0000E+00
 Initial X, Y emittances =  0.0000E+00,   0.0000E+00 m-rad
 Initial X,Y envelope radii  =  0.0000E+00,   0.0000E+00 m
 Initial X,Y envelope angles =  0.0000E+00,   0.0000E+00 rad
 Input beam current =  0.0000E+00 amps
 Current density =  0.0000E+00 amps/m**2
 Charge density =  0.0000E+00 Coul/m**3
 Number density =  0.0000E+00
 Plasma frequency     =  0.0000E+00 1/s
    times dt          =  0.0000E+00
    times quad period =  0.0000E+00
 Plasma period        =  6.2832E+36 s
 X-, Y-Thermal Velocities     =  0.0000E+00,   0.0000E+00 m/s
    times dt                  =  0.0000E+00,   0.0000E+00 m
    times dt/dx, dt/dy (X, Y) =  0.0000E+00,   0.0000E+00
 X-, Y-Debye Wavelengths  =  0.0000E+00,   0.0000E+00 m
    over dx, dy (X and Y) =  0.0000E+00,   0.0000E+00
 Longitudinal thermal velocity (rms) =  0.0000E+00 m/s
    times dt                   =  0.0000E+00 m
    times dt/dz                =  0.0000E+00
 Longitudinal Debye wavelength =  0.0000E+00 m
    over dz                    =  0.0000E+00
 Beam velocity =  0.0000E+00 m/s
    over c     =  0.0000E+00
 Kinetic energy =  0.0000E+00 eV
 Weight of simulation particles =  1.0000E+00
 Number of simulation particles =            0
 Number of real particles =  0.0000E+00
 Total mass =  0.0000E+00 kg
 Total charge =  0.0000E+00 Coul
 Generalized perveance =  0.0000E+00
 Characteristic current =  3.1297E+07 amps
 Budker parameter =  0.0000E+00
 Timestep size dt =  2.5000E-11 s
 Tune length =  0.0000E+00
 Undep. X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Undep. X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 Undep. X-, Y-Betatron wavelengths  =  0.0000E+00,   0.0000E+00 m
 Dep.   X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Dep.   X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 X-, Y-Tune Depressions (sigma/sigma0) =  0.0000E+00,   0.0000E+00
 Space charge wave velocity =  0.0000E+00 m/s
 Effective wall radius =  2.1355E-01 m
 Geometric factor =  7.9805E+01
 X-, Y-Emittance over Space charge forces =  0.0000E+00,   0.0000E+00
 Number of grid points in x =      200
 Number of grid points in y =      200
 Number of grid points in z =      100
 Grid spacing in x =  1.5100E-03 m
 Grid spacing in y =  1.2200E-03 m
 Grid spacing in z =  6.0200E-03 m
 Bend radius =  0.0000E+00 m
 Bending field =  0.0000E+00 T
 Bend length =  0.0000E+00 m
 Straight section length =  0.0000E+00 m
 Z at start of first bend =  0.0000E+00 m
it =      0 time = 0.0000E+00 pz = 0.0000E+00 ese = 0.0000E+00 ek = 0.0000E+00 et = 0.0000E+00
 ***  particle simulation package W3D running
Multigrid: Error converged to   2.778E-07 in     8 v-cycles             
it =      1 time = 2.5000E-11 pz = 0.0000E+00 ese = 7.1636E-12 ek = 0.0000E+00 et = 7.1636E-12
Multigrid: Error converged to   4.821E-07 in    14 v-cycles             
it =      2 time = 5.0000E-11 pz = 0.0000E+00 ese = 3.2275E-11 ek = 0.0000E+00 et = 3.2275E-11
/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py:62: UserWarning: there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics
  warnings.warn("there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics")
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 199, in __init__
    sec.add(incident_species = self.elecb.wspecies,
NameError: name 'sec' is not defined
--------------------------------------------------------------------------
mpirun noticed that the job aborted, but has no info as to the process
that caused that situation.
--------------------------------------------------------------------------
# Warp
# Origin date: Tue, 12 Nov 2019 15:31:14 -0800
# Local date: Tue, 12 Nov 2019 15:31:14 -0800
# Commit hash: 06ab50e
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warpCparallel.cpython-37m-x86_64-linux-gnu.so
# Thu Jan  2 19:50:50 2020
# 24 processors
# import warp time 1.6616742610931396 seconds
# For more help, type warphelp()
--------------------------------------------------------------------------
An MPI process has executed an operation involving a call to the
"fork()" system call to create a child process.  Open MPI is currently
operating in a condition that could result in memory corruption or
other system errors; your MPI job may hang, crash, or produce silent
data corruption.  The use of fork() (or system() or other calls that
create child processes) is strongly discouraged.  

The process that invoked fork was:

  Local host:          eu-a6-001-07 (PID 68486)
  MPI_COMM_WORLD rank: 3

If you are *absolutely sure* that your application will successfully
and correctly survive a call to fork(), you may disable this warning
by setting the mpi_warn_on_fork MCA parameter to 0.
--------------------------------------------------------------------------
('l_fftw_fort', False)
WARNING: module txphysics is not accessible.
WARNING: module desorb is not accessible.
0.0007224415584921976
Simulating ECLOUD in a crab cavity
[eu-a6-001-07:68478] 23 more processes have sent help message help-mpi-runtime.txt / mpi_init:warn-fork
[eu-a6-001-07:68478] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
Using cosine_3D emission angle distribution.
Secondary emission model: ECLOUD E0=150.0000 s=1.3500
#############################################################
Temp distribution found. Reloading it as initial distribution
#############################################################
Plot file name complete_sim.008.cgm
 ***  particle simulation package W3D generating
 ---  Resetting lattice array sizes
 ---  Allocating space for particles
 ---  Loading particles
 ---  Setting charge density
 ---  done
 ---  Allocating Win_Moments
 ---  Allocating Z_Moments
 ---  Allocating Lab_Moments
 Atomic number of ion =  1.0073E+00
 Charge state of ion  =  1.0000E+00
 Initial X, Y emittances =  0.0000E+00,   0.0000E+00 m-rad
 Initial X,Y envelope radii  =  0.0000E+00,   0.0000E+00 m
 Initial X,Y envelope angles =  0.0000E+00,   0.0000E+00 rad
 Input beam current =  0.0000E+00 amps
 Current density =  0.0000E+00 amps/m**2
 Charge density =  0.0000E+00 Coul/m**3
 Number density =  0.0000E+00
 Plasma frequency     =  0.0000E+00 1/s
    times dt          =  0.0000E+00
    times quad period =  0.0000E+00
 Plasma period        =  6.2832E+36 s
 X-, Y-Thermal Velocities     =  0.0000E+00,   0.0000E+00 m/s
    times dt                  =  0.0000E+00,   0.0000E+00 m
    times dt/dx, dt/dy (X, Y) =  0.0000E+00,   0.0000E+00
 X-, Y-Debye Wavelengths  =  0.0000E+00,   0.0000E+00 m
    over dx, dy (X and Y) =  0.0000E+00,   0.0000E+00
 Longitudinal thermal velocity (rms) =  0.0000E+00 m/s
    times dt                   =  0.0000E+00 m
    times dt/dz                =  0.0000E+00
 Longitudinal Debye wavelength =  0.0000E+00 m
    over dz                    =  0.0000E+00
 Beam velocity =  0.0000E+00 m/s
    over c     =  0.0000E+00
 Kinetic energy =  0.0000E+00 eV
 Weight of simulation particles =  1.0000E+00
 Number of simulation particles =            0
 Number of real particles =  0.0000E+00
 Total mass =  0.0000E+00 kg
 Total charge =  0.0000E+00 Coul
 Generalized perveance =  0.0000E+00
 Characteristic current =  3.1297E+07 amps
 Budker parameter =  0.0000E+00
 Timestep size dt =  2.5000E-11 s
 Tune length =  0.0000E+00
 Undep. X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Undep. X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 Undep. X-, Y-Betatron wavelengths  =  0.0000E+00,   0.0000E+00 m
 Dep.   X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Dep.   X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 X-, Y-Tune Depressions (sigma/sigma0) =  0.0000E+00,   0.0000E+00
 Space charge wave velocity =  0.0000E+00 m/s
 Effective wall radius =  2.1355E-01 m
 Geometric factor =  7.9805E+01
 X-, Y-Emittance over Space charge forces =  0.0000E+00,   0.0000E+00
 Number of grid points in x =      200
 Number of grid points in y =      200
 Number of grid points in z =      100
 Grid spacing in x =  1.5100E-03 m
 Grid spacing in y =  1.2200E-03 m
 Grid spacing in z =  6.0200E-03 m
 Bend radius =  0.0000E+00 m
 Bending field =  0.0000E+00 T
 Bend length =  0.0000E+00 m
 Straight section length =  0.0000E+00 m
 Z at start of first bend =  0.0000E+00 m
it =      0 time = 0.0000E+00 pz = 0.0000E+00 ese = 0.0000E+00 ek = 0.0000E+00 et = 0.0000E+00
 ***  particle simulation package W3D running
Multigrid: Error converged to   2.831E-07 in     8 v-cycles             
it =      1 time = 2.5000E-11 pz = 0.0000E+00 ese = 7.1675E-12 ek = 0.0000E+00 et = 7.1675E-12
Multigrid: Error converged to   4.821E-07 in    14 v-cycles             
it =      2 time = 5.0000E-11 pz = 0.0000E+00 ese = 3.0679E-11 ek = 0.0000E+00 et = 3.0679E-11
Saving a checkpoint!
===========================
Bunch passage: 7
Number of electrons in the dipole: 36050679
Number of macroparticles: 7820969
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 8
Number of electrons in the dipole: 43765960
Number of macroparticles: 9431976
Previous passage took 2826s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 8683141
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 21
Start SOFT regeneration. N_mp=8683141 Nel_tot=4.20e+07, En_tot = 3.63e+08
Applied correction factor = 4.342661e+00
Done SOFT regeneration. N_mp=1999509 Nel_tot=4.20e+07 En_tot=3.62e+08
Saving a checkpoint!
===========================
Bunch passage: 9
Number of electrons in the dipole: 56633843
Number of macroparticles: 5329079
Previous passage took 3176s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 10
Number of electrons in the dipole: 74498712
Number of macroparticles: 6129152
Previous passage took 1939s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 11
Number of electrons in the dipole: 99214805
Number of macroparticles: 7271691
Previous passage took 2166s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6548770
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 61
Start SOFT regeneration. N_mp=6548770 Nel_tot=1.23e+08, En_tot = 1.13e+09
Applied correction factor = 3.272709e+00
Done SOFT regeneration. N_mp=2001186 Nel_tot=1.23e+08 En_tot=1.13e+09
Saving a checkpoint!
===========================
Bunch passage: 12
Number of electrons in the dipole: 133232655
Number of macroparticles: 4319674
Previous passage took 2429s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 13
Number of electrons in the dipole: 179145932
Number of macroparticles: 5213953
Previous passage took 1706s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 14
Number of electrons in the dipole: 239576365
Number of macroparticles: 6230782
Previous passage took 1868s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 15
Number of electrons in the dipole: 316878468
Number of macroparticles: 7552958
Previous passage took 2106s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 7646884
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 202
Start SOFT regeneration. N_mp=7646884 Nel_tot=4.06e+08, En_tot = 3.93e+09
Applied correction factor = 3.820323e+00
Done SOFT regeneration. N_mp=2001026 Nel_tot=4.06e+08 En_tot=3.93e+09
Saving a checkpoint!
===========================
Bunch passage: 16
Number of electrons in the dipole: 412600445
Number of macroparticles: 3602040
Previous passage took 2343s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 17
Number of electrons in the dipole: 527013574
Number of macroparticles: 4414640
Previous passage took 1538s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 18
Number of electrons in the dipole: 658313677
Number of macroparticles: 5203475
Previous passage took 1636s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 19
Number of electrons in the dipole: 803046766
Number of macroparticles: 6107432
Previous passage took 1800s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 20
Number of electrons in the dipole: 957110008
Number of macroparticles: 7124048
Previous passage took 1931s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 7184996
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 555
Start SOFT regeneration. N_mp=7184996 Nel_tot=1.11e+09, En_tot = 1.19e+10
Applied correction factor = 3.589573e+00
Done SOFT regeneration. N_mp=2000709 Nel_tot=1.11e+09 En_tot=1.18e+10
Saving a checkpoint!
===========================
Bunch passage: 21
Number of electrons in the dipole: 1115879537
Number of macroparticles: 3055635
Previous passage took 2096s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 22
Number of electrons in the dipole: 1274741285
Number of macroparticles: 3707670
Previous passage took 1461s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 23
Number of electrons in the dipole: 1430477991
Number of macroparticles: 4241943
Previous passage took 1524s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 24
Number of electrons in the dipole: 1580394736
Number of macroparticles: 4736790
Previous passage took 1586s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 25
Number of electrons in the dipole: 1723180701
Number of macroparticles: 5246725
Previous passage took 1703s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
===========================
Bunch passage: 26
Number of electrons in the dipole: 1857152271
Number of macroparticles: 5756864
Previous passage took 1776s
/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py:62: UserWarning: there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics
  warnings.warn("there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics")
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.elecb.wspecies, self.nz,
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
IndexError: index 25000 is out of bounds for axis 0 with size 25000
--------------------------------------------------------------------------
mpirun noticed that the job aborted, but has no info as to the process
that caused that situation.
--------------------------------------------------------------------------
# Warp
# Origin date: Thu, 2 Jan 2020 15:21:51 -0800
# Local date: Thu, 2 Jan 2020 15:21:51 -0800
# Commit hash: d24b7d7
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warpCparallel.cpython-37m-x86_64-linux-gnu.so
# Sat Jan  4 21:57:59 2020
# 24 processors
# import warp time 2.0704774856567383 seconds
# For more help, type warphelp()
--------------------------------------------------------------------------
An MPI process has executed an operation involving a call to the
"fork()" system call to create a child process.  Open MPI is currently
operating in a condition that could result in memory corruption or
other system errors; your MPI job may hang, crash, or produce silent
data corruption.  The use of fork() (or system() or other calls that
create child processes) is strongly discouraged.  

The process that invoked fork was:

  Local host:          eu-a6-011-22 (PID 41629)
  MPI_COMM_WORLD rank: 0

If you are *absolutely sure* that your application will successfully
and correctly survive a call to fork(), you may disable this warning
by setting the mpi_warn_on_fork MCA parameter to 0.
--------------------------------------------------------------------------
('l_fftw_fort', False)
WARNING: module txphysics is not accessible.
WARNING: module desorb is not accessible.
0.0007224415584921976
Simulating ECLOUD in a crab cavity
[eu-a6-011-22:41624] 23 more processes have sent help message help-mpi-runtime.txt / mpi_init:warn-fork
[eu-a6-011-22:41624] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
Using cosine_3D emission angle distribution.
Secondary emission model: ECLOUD E0=150.0000 s=1.3500
Plot file name complete_sim.009.cgm
 ***  particle simulation package W3D generating
 ---  Resetting lattice array sizes
 ---  Allocating space for particles
 ---  Loading particles
 ---  Setting charge density
 ---  done
 ---  Allocating Win_Moments
 ---  Allocating Z_Moments
 ---  Allocating Lab_Moments
 Atomic number of ion =  1.0073E+00
 Charge state of ion  =  1.0000E+00
 Initial X, Y emittances =  0.0000E+00,   0.0000E+00 m-rad
 Initial X,Y envelope radii  =  0.0000E+00,   0.0000E+00 m
 Initial X,Y envelope angles =  0.0000E+00,   0.0000E+00 rad
 Input beam current =  0.0000E+00 amps
 Current density =  0.0000E+00 amps/m**2
 Charge density =  0.0000E+00 Coul/m**3
 Number density =  0.0000E+00
 Plasma frequency     =  0.0000E+00 1/s
    times dt          =  0.0000E+00
    times quad period =  0.0000E+00
 Plasma period        =  6.2832E+36 s
 X-, Y-Thermal Velocities     =  0.0000E+00,   0.0000E+00 m/s
    times dt                  =  0.0000E+00,   0.0000E+00 m
    times dt/dx, dt/dy (X, Y) =  0.0000E+00,   0.0000E+00
 X-, Y-Debye Wavelengths  =  0.0000E+00,   0.0000E+00 m
    over dx, dy (X and Y) =  0.0000E+00,   0.0000E+00
 Longitudinal thermal velocity (rms) =  0.0000E+00 m/s
    times dt                   =  0.0000E+00 m
    times dt/dz                =  0.0000E+00
 Longitudinal Debye wavelength =  0.0000E+00 m
    over dz                    =  0.0000E+00
 Beam velocity =  0.0000E+00 m/s
    over c     =  0.0000E+00
 Kinetic energy =  0.0000E+00 eV
 Weight of simulation particles =  1.0000E+00
 Number of simulation particles =            0
 Number of real particles =  0.0000E+00
 Total mass =  0.0000E+00 kg
 Total charge =  0.0000E+00 Coul
 Generalized perveance =  0.0000E+00
 Characteristic current =  3.1297E+07 amps
 Budker parameter =  0.0000E+00
 Timestep size dt =  2.5000E-11 s
 Tune length =  0.0000E+00
 Undep. X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Undep. X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 Undep. X-, Y-Betatron wavelengths  =  0.0000E+00,   0.0000E+00 m
 Dep.   X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Dep.   X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 X-, Y-Tune Depressions (sigma/sigma0) =  0.0000E+00,   0.0000E+00
 Space charge wave velocity =  0.0000E+00 m/s
 Effective wall radius =  2.1355E-01 m
 Geometric factor =  7.9805E+01
 X-, Y-Emittance over Space charge forces =  0.0000E+00,   0.0000E+00
 Number of grid points in x =      200
 Number of grid points in y =      200
 Number of grid points in z =      100
 Grid spacing in x =  1.5100E-03 m
 Grid spacing in y =  1.2200E-03 m
 Grid spacing in z =  6.0200E-03 m
 Bend radius =  0.0000E+00 m
 Bending field =  0.0000E+00 T
 Bend length =  0.0000E+00 m
 Straight section length =  0.0000E+00 m
 Z at start of first bend =  0.0000E+00 m
it =      0 time = 0.0000E+00 pz = 0.0000E+00 ese = 0.0000E+00 ek = 0.0000E+00 et = 0.0000E+00
 ***  particle simulation package W3D running
Multigrid: Error converged to   2.780E-07 in     8 v-cycles             
it =      1 time = 2.5000E-11 pz = 0.0000E+00 ese = 7.2505E-12 ek = 0.0000E+00 et = 7.2505E-12
Multigrid: Error converged to   4.849E-07 in    13 v-cycles             
it =      2 time = 5.0000E-11 pz = 0.0000E+00 ese = 2.9747E-11 ek = 0.0000E+00 et = 2.9747E-11
===========================
Bunch passage: 0
Number of electrons in the dipole: 19995557
Number of macroparticles: 4199067
mpirun: abort is already in progress...hit ctrl-c again to forcibly terminate

mpirun: killing job...

Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 73, in update_outputs
    elecs_density_tot = (sw.get_density(l_dividebyvolume=0)[:,:,:] 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/particles/species.py", line 601, in get_density
    density = fzeros([nx+1, ny+1, nz+1], 'd')
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/Forthon/_Forthon.py", line 349, in fzeros
    return zeros(shape, order='F', *args)
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 73, in update_outputs
    elecs_density_tot = (sw.get_density(l_dividebyvolume=0)[:,:,:] 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/particles/species.py", line 601, in get_density
    density = fzeros([nx+1, ny+1, nz+1], 'd')
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/Forthon/_Forthon.py", line 349, in fzeros
    return zeros(shape, order='F', *args)
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 73, in update_outputs
    elecs_density_tot = (sw.get_density(l_dividebyvolume=0)[:,:,:] 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/particles/species.py", line 601, in get_density
    density = fzeros([nx+1, ny+1, nz+1], 'd')
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/Forthon/_Forthon.py", line 349, in fzeros
    return zeros(shape, order='F', *args)
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 73, in update_outputs
    elecs_density_tot = (sw.get_density(l_dividebyvolume=0)[:,:,:] 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/particles/species.py", line 601, in get_density
    density = fzeros([nx+1, ny+1, nz+1], 'd')
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/Forthon/_Forthon.py", line 349, in fzeros
    return zeros(shape, order='F', *args)
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 288, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 72, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,int(nz/2.)])
KeyboardInterrupt
# Warp
# Origin date: Thu, 2 Jan 2020 15:21:51 -0800
# Local date: Thu, 2 Jan 2020 15:21:51 -0800
# Commit hash: d24b7d7
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warpCparallel.cpython-37m-x86_64-linux-gnu.so
# Sat Jan  4 22:04:00 2020
# 24 processors
# import warp time 2.290421724319458 seconds
# For more help, type warphelp()
--------------------------------------------------------------------------
An MPI process has executed an operation involving a call to the
"fork()" system call to create a child process.  Open MPI is currently
operating in a condition that could result in memory corruption or
other system errors; your MPI job may hang, crash, or produce silent
data corruption.  The use of fork() (or system() or other calls that
create child processes) is strongly discouraged.  

The process that invoked fork was:

  Local host:          eu-a6-007-21 (PID 41830)
  MPI_COMM_WORLD rank: 2

If you are *absolutely sure* that your application will successfully
and correctly survive a call to fork(), you may disable this warning
by setting the mpi_warn_on_fork MCA parameter to 0.
--------------------------------------------------------------------------
('l_fftw_fort', False)
WARNING: module txphysics is not accessible.
WARNING: module desorb is not accessible.
0.0007224415584921976
Simulating ECLOUD in a crab cavity
[eu-a6-007-21:41823] 23 more processes have sent help message help-mpi-runtime.txt / mpi_init:warn-fork
[eu-a6-007-21:41823] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
Using cosine_3D emission angle distribution.
Secondary emission model: ECLOUD E0=150.0000 s=1.3500
Plot file name complete_sim.010.cgm
 ***  particle simulation package W3D generating
 ---  Resetting lattice array sizes
 ---  Allocating space for particles
 ---  Loading particles
 ---  Setting charge density
 ---  done
 ---  Allocating Win_Moments
 ---  Allocating Z_Moments
 ---  Allocating Lab_Moments
 Atomic number of ion =  1.0073E+00
 Charge state of ion  =  1.0000E+00
 Initial X, Y emittances =  0.0000E+00,   0.0000E+00 m-rad
 Initial X,Y envelope radii  =  0.0000E+00,   0.0000E+00 m
 Initial X,Y envelope angles =  0.0000E+00,   0.0000E+00 rad
 Input beam current =  0.0000E+00 amps
 Current density =  0.0000E+00 amps/m**2
 Charge density =  0.0000E+00 Coul/m**3
 Number density =  0.0000E+00
 Plasma frequency     =  0.0000E+00 1/s
    times dt          =  0.0000E+00
    times quad period =  0.0000E+00
 Plasma period        =  6.2832E+36 s
 X-, Y-Thermal Velocities     =  0.0000E+00,   0.0000E+00 m/s
    times dt                  =  0.0000E+00,   0.0000E+00 m
    times dt/dx, dt/dy (X, Y) =  0.0000E+00,   0.0000E+00
 X-, Y-Debye Wavelengths  =  0.0000E+00,   0.0000E+00 m
    over dx, dy (X and Y) =  0.0000E+00,   0.0000E+00
 Longitudinal thermal velocity (rms) =  0.0000E+00 m/s
    times dt                   =  0.0000E+00 m
    times dt/dz                =  0.0000E+00
 Longitudinal Debye wavelength =  0.0000E+00 m
    over dz                    =  0.0000E+00
 Beam velocity =  0.0000E+00 m/s
    over c     =  0.0000E+00
 Kinetic energy =  0.0000E+00 eV
 Weight of simulation particles =  1.0000E+00
 Number of simulation particles =            0
 Number of real particles =  0.0000E+00
 Total mass =  0.0000E+00 kg
 Total charge =  0.0000E+00 Coul
 Generalized perveance =  0.0000E+00
 Characteristic current =  3.1297E+07 amps
 Budker parameter =  0.0000E+00
 Timestep size dt =  2.5000E-11 s
 Tune length =  0.0000E+00
 Undep. X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Undep. X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 Undep. X-, Y-Betatron wavelengths  =  0.0000E+00,   0.0000E+00 m
 Dep.   X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Dep.   X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 X-, Y-Tune Depressions (sigma/sigma0) =  0.0000E+00,   0.0000E+00
 Space charge wave velocity =  0.0000E+00 m/s
 Effective wall radius =  2.1355E-01 m
 Geometric factor =  7.9805E+01
 X-, Y-Emittance over Space charge forces =  0.0000E+00,   0.0000E+00
 Number of grid points in x =      200
 Number of grid points in y =      200
 Number of grid points in z =      100
 Grid spacing in x =  1.5100E-03 m
 Grid spacing in y =  1.2200E-03 m
 Grid spacing in z =  6.0200E-03 m
 Bend radius =  0.0000E+00 m
 Bending field =  0.0000E+00 T
 Bend length =  0.0000E+00 m
 Straight section length =  0.0000E+00 m
 Z at start of first bend =  0.0000E+00 m
it =      0 time = 0.0000E+00 pz = 0.0000E+00 ese = 0.0000E+00 ek = 0.0000E+00 et = 0.0000E+00
 ***  particle simulation package W3D running
Multigrid: Error converged to   2.794E-07 in     8 v-cycles             
it =      1 time = 2.5000E-11 pz = 0.0000E+00 ese = 7.4932E-12 ek = 0.0000E+00 et = 7.4932E-12
Multigrid: Error converged to   4.849E-07 in    13 v-cycles             
it =      2 time = 5.0000E-11 pz = 0.0000E+00 ese = 3.1298E-11 ek = 0.0000E+00 et = 3.1298E-11
===========================
Bunch passage: 0
Number of electrons in the dipole: 19995023
Number of macroparticles: 4198955
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 1
Number of electrons in the dipole: 20752668
Number of macroparticles: 4382240
Previous passage took 1660s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 2
Number of electrons in the dipole: 21997248
Number of macroparticles: 4627231
Previous passage took 1720s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 3
Number of electrons in the dipole: 23478090
Number of macroparticles: 5030058
Previous passage took 1772s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 4
Number of electrons in the dipole: 25327254
Number of macroparticles: 5527242
Previous passage took 1830s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 5
Number of electrons in the dipole: 27913494
Number of macroparticles: 6160989
Previous passage took 1895s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 6
Number of electrons in the dipole: 31564417
Number of macroparticles: 7094438
Previous passage took 1977s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6188501
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 13
Start SOFT regeneration. N_mp=6188501 Nel_tot=2.65e+07, En_tot = 2.73e+08
Applied correction factor = 3.094727e+00
Done SOFT regeneration. N_mp=1999481 Nel_tot=2.65e+07 En_tot=2.73e+08
Saving a checkpoint!
===========================
Bunch passage: 7
Number of electrons in the dipole: 36674640
Number of macroparticles: 4136736
Previous passage took 2106s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 8
Number of electrons in the dipole: 43698995
Number of macroparticles: 4787889
Previous passage took 1624s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 9
Number of electrons in the dipole: 53269102
Number of macroparticles: 5559431
Previous passage took 1720s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 10
Number of electrons in the dipole: 66418883
Number of macroparticles: 6616228
Previous passage took 1893s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6655244
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 38
Start SOFT regeneration. N_mp=6655244 Nel_tot=7.79e+07, En_tot = 7.78e+08
Applied correction factor = 3.329788e+00
Done SOFT regeneration. N_mp=1998729 Nel_tot=7.79e+07 En_tot=7.79e+08
Saving a checkpoint!
===========================
Bunch passage: 11
Number of electrons in the dipole: 84502614
Number of macroparticles: 3381303
Previous passage took 2110s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 12
Number of electrons in the dipole: 109199959
Number of macroparticles: 4181899
Previous passage took 1563s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 13
Number of electrons in the dipole: 142617745
Number of macroparticles: 5095091
Previous passage took 1709s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 14
Number of electrons in the dipole: 187336349
Number of macroparticles: 6302250
Previous passage took 1919s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6904868
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 120
Start SOFT regeneration. N_mp=6904868 Nel_tot=2.41e+08, En_tot = 2.36e+09
Applied correction factor = 3.455980e+00
Done SOFT regeneration. N_mp=1998017 Nel_tot=2.41e+08 En_tot=2.36e+09
Saving a checkpoint!
===========================
Bunch passage: 15
Number of electrons in the dipole: 246137393
Number of macroparticles: 2986551
Previous passage took 2156s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 16
Number of electrons in the dipole: 321781242
Number of macroparticles: 3789077
Previous passage took 1554s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 17
Number of electrons in the dipole: 416178910
Number of macroparticles: 4661396
Previous passage took 1696s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 18
Number of electrons in the dipole: 529406244
Number of macroparticles: 5734517
Previous passage took 1875s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6301373
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 328
Start SOFT regeneration. N_mp=6301373 Nel_tot=6.57e+08, En_tot = 6.64e+09
Applied correction factor = 3.151778e+00
Done SOFT regeneration. N_mp=1998901 Nel_tot=6.57e+08 En_tot=6.64e+09
Saving a checkpoint!
===========================
Bunch passage: 19
Number of electrons in the dipole: 660034170
Number of macroparticles: 2724903
Previous passage took 2086s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 20
Number of electrons in the dipole: 804808338
Number of macroparticles: 3435802
Previous passage took 1535s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 21
Number of electrons in the dipole: 959063642
Number of macroparticles: 4090031
Previous passage took 1605s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 22
Number of electrons in the dipole: 1118181796
Number of macroparticles: 4792052
Previous passage took 1721s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 23
Number of electrons in the dipole: 1278036849
Number of macroparticles: 5543243
Previous passage took 1858s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 24
Number of electrons in the dipole: 1434975814
Number of macroparticles: 6331354
Previous passage took 1938s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6680426
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 791
Start SOFT regeneration. N_mp=6680426 Nel_tot=1.58e+09, En_tot = 1.77e+10
Applied correction factor = 3.341897e+00
Done SOFT regeneration. N_mp=1999383 Nel_tot=1.58e+09 En_tot=1.77e+10
Saving a checkpoint!
===========================
Bunch passage: 25
Number of electrons in the dipole: 1586024189
Number of macroparticles: 2466252
Previous passage took 2039s
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py:62: UserWarning: there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics
  warnings.warn("there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics")
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_0.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    self.step(self.tot_nsteps-self.n_step)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.secelec.wspecies, self.b_pass)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    density = False)
IndexError: index 25 is out of bounds for axis 0 with size 25
--------------------------------------------------------------------------
mpirun noticed that the job aborted, but has no info as to the process
that caused that situation.
--------------------------------------------------------------------------
